clang/test/CXX/class/class.mem/p13.cpp

```diff
@@ -63,7 +63,7 @@ struct X4 { // expected-note{{previous}}
     int X;
     union {
       float Y;
-      unsigned X4; // expected-error{{redeclares 'X4'}}
+      unsigned X4; // expected-error{{redeclares 'X4'}} expected-error {{'X4' has the same name as its class}}
```

Comment:
I'm not sure it is good to have that type of double erroring,

Reply:
Oof. Each are correct, each are helpful in isolation, but having both show up at the same time is a bit chatty. I think the "redeclares" diagnostic is the more informative of the two so I weakly prefer keeping that one and suppressing the other, but I think it's defensible either way.